.prettierrc.js

```diff
@@ -0,0 +1 @@
+module.exports = {};
```

Comment:
We have a standard prettier config that most of other apps use - e.g https://github.com/customink/ci-spa-starter/blob/main/templates/react-query/.prettierrc.js